operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -7,6 +7,35 @@
 # the pod should be restarted. The script checks a WebLogic Server state file which
 # is updated by the node manager.
 
+function newCopySitCfg() {
+  (
+  shopt -s nullglob  # force file matching 'glob' for loops below to run 0 times if 0 matches
```

Comment:
no unset of the nullglob may have side effect

Reply:
@tbarnes-us any comments on this?

Reply:
The nullglob is scoped to the surrounding "(" and ")".   @jshum2479

Reply:
example output from

Reply:
```
fn ()  {
  (
  shopt -s nullglob
  echo "--start fn--"
  for file in doesnotexist*; do
    echo fileinfn: $file
  done
  echo "--end fn--"
  )
}

for file in doesnotexist*; do
  echo fileinmain1: $file
done
fn

for file in doesnotexist*; do
  echo fileinmain2: $file
done

shopt -s nullglob

for file in doesnotexist*; do
  echo fileinmain3: $file
done
```

Reply:
```
fileinmain1: doesnotexist*
--start fn--
--end fn--
fileinmain2: doesnotexist*
```